apps/web/src/components/widget/NotificationCenterWidget.tsx

```diff
@@ -41,6 +41,7 @@ function PopoverWrapper() {
       colorScheme={colorScheme}
       onNotificationClick={handlerOnNotificationClick}
       onActionClick={handlerOnActionClick}
+      position="bottom-end"
```

Comment:
@BiswaViraj  Can we make `bottom-end` the default position? So that we won't have to specify this prop for default behaviour.

Reply:
@scopsy, yeah we can have `bottom-end` as default instead of `bottom`